packages/kununu-utils/kununu-logger/fingers-crossed/index.js

```diff
@@ -0,0 +1,117 @@
+import {formatNodeRequest} from '../index';
+
+const TransportStream = require('winston-transport'); // eslint-disable-line import/no-extraneous-dependencies
+
+module.exports = class FingersCrossed extends TransportStream {
+  constructor (options = {}) {
+    super(options);
+
+    this.name = options.name || 'fingers-crossed';
+    this.level = options.level || 'info';
+    this.activationLogLevel = options.activationLogLevel || 'error';
+    this.levels = options.levels;
+
+    this.state = [];
```

Comment:
When is this being cleaned? Isn't this going to grow forever?

Reply:
Is it being cleaned on `stateRemove` (just renamed to `cleanState` on a new commit to make this more clear). We have unit tests that cover it, something like this:
1. info, trace-id-1 = expect not to output (saves on state)
2. info, trace-id-1 = expect not to output (saves on state)
3. error, trace-id-1 = expect to output 1) 2) 3) on console and clear logs with this same trace id
4. error, trace-id-1 = expect to output only itself as there are no other logs with this same trace id